src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest_cancelCourtHearing.java

```diff
@@ -42,7 +42,7 @@ public Clock clock() {
 
     @Before
     public void setup() {
-        token = authTokenHelper.getToken(AuthTokenHelper.AuthToken.NORMAL_USER);
+        token = authTokenHelper.getToken(AuthTokenHelper.AuthToken.COURT_HEARING_MAINTAINER);
```

Comment:
would be good to test the negation too i.e. I could delete all of the `PreAuthorize` lines from the code and all the tests would still pass

Reply:
Good point.  Will add.